src/main/java/com/softserve/actent/controller/CityController.java

```diff
@@ -0,0 +1,96 @@
+package com.softserve.actent.controller;
+
+import com.softserve.actent.model.dto.CityDto;
+import com.softserve.actent.model.dto.IdDto;
+import com.softserve.actent.model.dto.converter.CityConverter;
+import com.softserve.actent.model.entity.City;
+import com.softserve.actent.service.CityService;
+import com.softserve.actent.service.RegionService;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
```

Comment:
read about RestController vs Controler annotation. Do you still need to return ResponseEntity if you use RestController?

Reply:
ResponseEntity is just more flexible — you may control what goes in response(body, headers and status code) in method. Maybe, we should check, where we have no need in this(to be honestly, almost everywhere) and remove it(a lot of refactorings, meh).